tldextract/tldextract.py

```diff
@@ -259,12 +263,12 @@ def _get_cached_tld_extractor(self):
         try:
             with open(self.cache_file, 'rb') as cache_file:
                 try:
-                    suffixes = pickle.load(cache_file)
-                except Exception as myriad_unpickling_errors: # pylint: disable=broad-except
+                    suffixes = json.load(cache_file)
+                except Exception as exc: # pylint: disable=broad-except
```

Comment:
Are `json.load`'s exceptions any easier to enumerate? Maybe `(IOError, ValueError)`?